app/views/dashboards/show.html.erb

```diff
@@ -214,14 +214,11 @@
                 <li class="nav-item active">
                   <%= link_to 'Dashboard', root_path, class: 'nav-link' %>
                 </li>
-                <li class="nav-item">
-                  <%= link_to 'Members', members_path, class: 'nav-link' %>
-                </li>
                 <li class="nav-item">
                   <%= link_to 'Events', events_path, class: 'nav-link' %>
                 </li>
                 <li class="nav-item">
-                  <%= link_to 'Leaderboard', events_path, class: 'nav-link' %>
+                  <%= link_to 'Leaderboard', members_path, class: 'nav-link' %>

```

Comment:
We should change all the other navbars to reflect this change, much cleaner good for usability